test_rclcpp/test/test_local_parameters.cpp

```diff
@@ -196,7 +197,7 @@ TEST_F(CLASSNAME(test_local_parameters, RMW_IMPLEMENTATION), helpers) {
     rclcpp::Parameter("baz", 1.45),
     rclcpp::Parameter("foobar", true),
     rclcpp::Parameter("barfoo", std::vector<uint8_t>{0, 1, 2}),
-  });
+  }, 1s);
```

Comment:
see [this comment](https://github.com/ros2/rclcpp/issues/360#issue-252127177), 1 second should be long enough.